packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -133,10 +132,26 @@ export default class SearchBox extends Base {
    * Invoked in response to a user clicking inside of the searchbox input.
    */
   clickExposed() {
-    const searchBoxClickedEvent = new CustomEvent(SEARCHBOX_EVENT.SEARCHBOX_CLICK, { bubbles: true });
+    const searchBoxClickedEvent = this.getCustomEvent(SEARCHBOX_EVENT.SEARCHBOX_CLICK);
     this.dispatchEvent(searchBoxClickedEvent);
   }
 
+  /**
+   * Returns an event with a standard structure.
+   *
+   * @param type The type (or name) of the event to be emitted.
+   * @param detail A payload to be sent with the event.
+   */
+  getCustomEvent(type: string, detail: object = {}) {
```

Comment:
I would call this `createCustomEvent`.

Reply:
I think `detail` should be optional. Attempting to spread `undefined` will succeed and expand to nothing.